MAUI/LinearGauge/mirror-linear-gauge.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Mirror in .NET MAUI Linear Gauge control | Syncfusion
+description: Learn here about mirroring the Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control with IsMirrored property
```

Comment:
Learn how to use the `IsMirrored` property to mirror the Syncfusion.NET MAUI Linear Gauge (SfLinearGauge) control.